tcms/testplans/static/testplans/js/get.js

```diff
@@ -34,8 +36,91 @@ $(document).ready(function() {
     });
 
     collapseDocumentText();
+    initTestCaseSearchAndAdd(testPlanId);
 });
 
+
+function addTestCaseToPlan(planId) {
+    const case_name = $('#search-testcase')[0].value;
+    const test_case = autocomplete_cache[case_name];
```

Comment:
`case_name`, `test_case` -> `caseName`, `testCase`. otherwise we will have more work later when we start linting this